dist/components/tabs/stencil-tabs.js

```diff
@@ -0,0 +1,121 @@
+$(document).ready(function() {
+    var init = function() {
+        // Set up initial state for accessibility
+        var $tabButtons = $('.c-tabs__button');
```

Comment:
Roger that. In fact, @ry5n you've definitely explained this to me before. I'll try and keep a lid on this particular idea for the future. Sorry for the repeat question.